index.html

```diff
@@ -53,6 +53,7 @@
                 <br>
             </div>
             <h3><a id="zen-video-title"></a></h3>
+            <div id="zen-video-description"></div>
```

Comment:
Let's move this down between #playerControls and #demo, and probably want a `<br>` above & below, including the one that's already there